test/gtest/ucp/ucp_test.cc

```diff
@@ -114,6 +114,16 @@ bool ucp_test::has_any_transport(const std::vector<std::string>& tl_names) const
            all_tl_names.end();
 }
 
+bool ucp_test::has_any_transport(const std::string *tls, size_t tl_size) const {
+    const std::vector<std::string> tl_names =
+        std::vector<std::string>(tls, tls + tl_size);
+    const std::vector<std::string>& all_tl_names = GetParam().transports;
+
+    return std::find_first_of(all_tl_names.begin(), all_tl_names.end(),
+                              tl_names.begin(),     tl_names.end()) !=
+           all_tl_names.end();
```

Comment:
maybe initialize std::vector and pass it to `ucp_test::has_any_transport(const std::vector<std::string>& tl_names)`